.classpath

```diff
@@ -34,7 +34,6 @@
 	<classpathentry kind="lib" path="/usr/share/java/junit.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/ldapjdk.jar"/>
 	<classpathentry kind="lib" path="/usr/share/java/jaxb-api.jar"/>
```

Comment:
Should this get dropped as well? If we're not pulling in jaxb-api by default?

Reply:
Can Eclipse build the code on F33 without `jaxb-api.jar`? I thought at least the API is required.

Reply:
I wasn't sure if you were trying to build a F31/F32-only elcipse (since they're the only released platform) -- or if you were trying to make it compatible with all of them. I guess only the API should be necessary for build.... can't Eclipse default to `cmake` for builds though? Or are you doing something special?

Reply:
Not sure if Eclipse can use cmake. Usually I just create an Eclipse project and point it to PKI source folder which already has the `.classpath` and `.project` files. We want these files to work with as many platforms as possible, but mainly F31-F33.

Reply:
It looks like we don't need to do this actually:

 - https://stackoverflow.com/questions/14810849/why-does-m2e-plugin-for-eclipse-insert-optional-attribute-to-src-and-what-does-i
 - https://bugs.eclipse.org/bugs/show_bug.cgi?id=388541

I think we can fix this by using:

```
<classpathentry kind="lib" path="....">
    <attributes>
        <attribute name="optional" value="true"/>
    </attributes>
</classpathentry>
```

Does that fix your IDE error perhaps? :)